client/ui/actions/addon-groups.js

```diff
@@ -173,3 +190,16 @@ export function updateAddonGroup(pk, data, saveAndContinue) {
     });
   };
 }
+
+export function syncAddonGroup(pk) {
+  return dispatch => {
+    dispatch(requestSyncAddonGroup(pk));
```

Comment:
`requestSyncAddonGroup` is defined on line 114, right? It doesn't take any arguments.